pkg/controller/razeedeployment/razeedeployment_controller.go

```diff
@@ -841,21 +986,71 @@ func (r *ReconcileRazeeDeployment) finalizeRazeeDeployment(req *marketplacev1alp
 	reqLogger.Info("Uninstall job created successfully")
 	reqLogger.Info("Successfully finalized RazeeDeployment")
 
-	// Remove the razeeDeploymentFinalizer
+	// Remove the RAZEE_DEPLOYMENT_FINALIZER
 	// Once all finalizers are removed, the object will be deleted
-	req.SetFinalizers(utils.RemoveKey(req.GetFinalizers(), razeeDeploymentFinalizer))
+	req.SetFinalizers(utils.RemoveKey(req.GetFinalizers(), utils.RAZEE_DEPLOYMENT_FINALIZER))
 	err = r.client.Update(context.TODO(), req)
 	if err != nil {
 		return reconcile.Result{}, err
 	}
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileRazeeDeployment) MakeRazeeJob(
```

Comment:
Regression on this - check dev. Code is diff.

Reply:
Hey @zach-source I think this one matches up. I'm just reading the razee job name off the constant in utils, targetNamespace comes from the instance, as does the file source url. .

Reply:
Yep, you're right, my fault on this one.